lib/auth/tls_test.go

```diff
@@ -3888,20 +3892,65 @@ func TestGRPCServer_CreateTokenV2(t *testing.T) {
 
 		requireTokenCreated bool
 		requireError        require.ErrorAssertionFunc
-		requireEventEmitted bool
+		auditEvents         []eventtypes.AuditEvent
```

Comment:
Obviously this is not your choice @bl-nero, but does it make sense for this test to be in a file named `tls_test.go`? Unless I'm mistaken, this has nothing to do with a "TLS test".

Reply:
Agreed - it's out of scope for this PR - but I'd like to eventually see `tls_test.go` dissolve into more appropriate files (e.g `grpcserver_test.go`). I'll probably just pursue a refactor PR later this week that moves each of these tests to their appropriate home.